Node.gitignore

```diff
@@ -70,7 +70,13 @@ typings/
 .vuepress/dist
 
 # Serverless directories
-.serverless
+.serverless/
 
 # FuseBox cache
 .fusebox/
+
+#DynamoDB Local files 
+.dynamodb/ 
+
+#VS Code editor files 
```

Comment:
Thanks, but already have a template at [`Global/VisualStudioCode.gitignore`](https://github.com/github/gitignore/blob/master/Global/VisualStudioCode.gitignore) so I'd like to avoid duplicating rules. Please drop it from the PR.